### PR TITLE
Development on env

### PR DESCRIPTION
**Pull request by wenxinnnnn**
1.Complete the curd of env.
2.Complete the global env switch.

### CODE REVIEW

File: dashboard/yarn.lock [hunk 1]
```diff
@@ -0,0 +1,17632 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```
Comment (wonderflow) [inline dashboard/yarn.lock:2]: Can we remove this file?

Comment (wenxinnnnn) [reply dashboard/yarn.lock:2]: ok！I will delete yarn.lock , and add to the ignore file

**Comment by zzxwill** [comment]
The whole design is as expected, great.
BTW, it would be better to combine those 13 commits into one or two.

**Comment by zzxwill** [comment]
Closed it as it has been merged in #230

**Pull request closed without merging**